internal/console/pre_scan.go

```diff
@@ -50,15 +50,23 @@ func preRun(cmd *cobra.Command) error {
 
 func setupConfigFile() (bool, error) {
 	if flags.GetStrFlag(flags.ConfigFlag) == "" {
+		configPath := ""
 		path := flags.GetMultiStrFlag(flags.PathFlag)
-		if len(path) == 0 {
-			return true, nil
-		}
 		if len(path) > 1 {
 			warnings = append(warnings, "Any kics.config file will be ignored, please use --config if kics.config is wanted")
 			return true, nil
 		}
-		configPath := path[0]
+
+		if len(path) == 1 {
+			configPath = path[0]
+		}
+		if configPath == "" {
```

Comment:
@lipeavelar, it seems that `configPath == ""` will only happen if the user points to multiple paths. But what if the user points to one path but the `config.file` is not in the project directory?

Sorry if I am not understanding correctly.

Reply:
In this case, it will preserve the old behavior, so it will not use config file, like KICS already does today. In this case, the user should use `--config` and give the file explicitly...